src/Microsoft.AspNetCore.Mvc.Analyzers/SymbolNames.cs

```diff
@@ -58,5 +58,9 @@ internal static class SymbolNames
         public const string ProducesResponseTypeAttribute = "Microsoft.AspNetCore.Mvc.ProducesResponseTypeAttribute";
 
         public const string RenderPartialMethod = "RenderPartial";
+
+        public const string RequiredAttribute = "RequiredAttribute";
+
+        public const string CollectionType= "SourceComplexParameterSymbol";
```

Comment:
Doesn't this symbol both exclude `SourceComplexParameterSymbolWithCustomModifiers` and indicate only that the parameter is not simple e.g. that it has associated attributes? That is, what about this type indicates a collection?

@pranavkm

Reply:
I don't think you're using this anymore.